ReactiveXComponentTest/WebSocket/WebSocketMessageHelperTest.cs

```diff
@@ -0,0 +1,190 @@
+﻿using System;
+using System.IO;
+using System.Text;
+using NFluent;
+using NUnit.Framework;
+using ReactiveXComponent.Common;
+using ReactiveXComponent.WebSocket;
+
+namespace ReactiveXComponentTest.WebSocket
+{
+    [TestFixture]
+    public class WebsocketMessageHelperTest
+    {
+        private int _helloworldComponent;
+        private string _message;
+        private string _topic;
+        private WebSocketEngineHeader _webSocketEngineHeader;
+        private WebSocketPacket _webSocketPacket;
+
+        [SetUp]
+        public void Setup()
+        {
+            _helloworldComponent = -69981087;
+            const long helloworldStateMachine = -829536631;
+
+            _message = "HelloWorld";
+            _topic = "HelloWorldTopic";
+            _webSocketEngineHeader = new WebSocketEngineHeader
+            {
+                ComponentCode = new Option<long>(_helloworldComponent),
+                StateMachineCode = new Option<long>(helloworldStateMachine),
+                MessageType = new Option<string>(_message.GetType().ToString()),
+            };
+            _webSocketPacket = new WebSocketPacket
+            {
+                Header = _webSocketEngineHeader,
+                JsonMessage = SerializeToString(_message),
+            };
+        }
+
+        [TestCase(WebSocketCommand.Input)]
+        [TestCase(WebSocketCommand.Subscribe)]
+        [TestCase(WebSocketCommand.Unsubscribe)]
+        [TestCase(WebSocketCommand.Snapshot)]
+        [TestCase(WebSocketCommand.Error)]
+        [TestCase(WebSocketCommand.Update)]
+        [TestCase("")]
+        public void SerializeRequest_SuccessfulSerialization_Test(string command)
+        {
+            var request = WebSocketMessageHelper.SerializeRequest(command, _webSocketEngineHeader, _message,
+                _helloworldComponent.ToString(), _topic);
+
+            var header = SerializeToString(_webSocketPacket);
+            var expectedRequest = $"{command} {_topic} {_helloworldComponent} {header}{Environment.NewLine}";
+
+            Check.That(request.Equals(expectedRequest)).IsTrue();
+        }
+
+        [TestCase(WebSocketCommand.Input)]
+        [TestCase(WebSocketCommand.Subscribe)]
```

Comment:
Am not sure that all these commands expect a request built with a *command* a *topic* and a *component* *code*. For instance the _Subscribe_ command is simply built this way: "_subscribe_ _{json}_"